### ISSUE TITLE
prisma/codemods doesn't transform anything

### ISSUE THREAD

**Issue by pasekit** [prisma/codemods#22]
**Describe the bug**
I tried to upgrade from Prisma 2.10.1 to 2.15.0 in my NestJs project (7.6.5) with `npx @prisma/codemods update-2.12 ./ ` To do this I tried to change the breaking changes with the codemods. However, codemods reports that all three transforms has not changed or skipped any files. 
**Before the codemod**
```
		let output = await this.prisma.genUser.findOne({
			where
		});
```

**After the codemod**
```
		let output = await this.prisma.genUser.findOne({
			where
		});
```

**Expected output**
```
		let output = await this.prisma.genUser.findUnique({
			where
		});
```

**Additional context**
these are parts of my project tree:

├───prisma
├───src
│   ├───apps
│   │   ├───general
│   │   │   ├───area
│   │   │   ├───auth
│   │   │   │   ├───guards
│   │   │   │   └───strategies
│   │   │   ├───log
│   │   │   ├───login
│   │   │   ├───organization
│   │   │   ├───user

**Comment by williamluke4** [comment]
Hey @pasekit, Thanks for reporting this, I've added a test case for this [here](https://github.com/prisma/codemods/blob/main/tests/__fixtures__/findUnique/inputs/minimal.ts#L16-L20). 

Could you try running this again using the following (i.e 0.6.4-next.2)
```
npx @prisma/codemods@next update-2.12 ./
```
Let me know if this solves your problem :)

**Comment by pasekit** [comment]
did the update to version 0.7.0 but same problem. nothing changed..

    PS C:\Apps\PASEK\Backend> npx @prisma/codemods update-2.12 ./
    Executing command: jscodeshift --verbose=0 --no-babel --ignore-config=.gitignore --ignore-pattern=**/node_modules/** --    extensions=tsx,ts,js,tsx --parser=tsx --transform C:\Apps\PASEK\Backend\node_modules\@prisma\codemods\transforms\namespace.js ./
    No files selected, nothing to do.
    All done.
    Results:
    0 errors
    0 unmodified
    0 skipped
    0 ok
    Time elapsed: 0.034seconds
    Executing command: jscodeshift --verbose=0 --no-babel --ignore-config=.gitignore --ignore-pattern=**/node_modules/** --extensions=tsx,ts,js,tsx --parser=tsx --transform     C:\Apps\PASEK\Backend\node_modules\@prisma\codemods\transforms\findUnique.js ./
    No files selected, nothing to do.
    All done.
    Results:
    0 errors
    0 unmodified
    0 skipped
    0 ok
    Time elapsed: 0.034seconds
    Executing command: jscodeshift --verbose=0 --no-babel --ignore-config=.gitignore --ignore-pattern=**/node_modules/** --    extensions=tsx,ts,js,tsx --parser=tsx --transform C:\Apps\PASEK\Backend\node_modules\@prisma\codemods\transforms\to$.js ./
    No files selected, nothing to do.
    All done.
    Results:
    0 errors
    0 unmodified
    0 skipped
    0 ok
    Time elapsed: 0.034seconds

**Comment by williamluke4** [comment]
Could you then provide a more complete reproduction as the test cases are [passing](https://github.com/prisma/codemods/blob/main/tests/__snapshots__/findUnique/input/minimal.ts#L20)!

**Comment by pasekit** [comment]
i was able to solve the problem by specifying the subfolder where the files are located. The files were not recognized on the top level nor on the ./src level. Only when I specified ./src/apps, the files were recognized. But then I had to specify the schema file, because it was located under ./prisma/schema.prisma.

the command looked like this:
npx @prisma/codemods update-2.12 .\src\apps --schemaPath .\prisma\schema.prisma
